### ISSUE TITLE
How can I get the PRE + PNR + POST frames 

### ISSUE THREAD

**Issue by NNNNAI** [EGO4D/hands-and-objects#6]
As you mention in the readme.md, training these model need PRE + PNR + POST frames . Could you tell me how can I get these frames by using cli ? Thanks~

**Comment by Sid2697** [comment]
Hello @NNNNAI, thank you for reaching out!

Using the CLI, you'll be able to download the dataset and the corresponding annotations. This link (https://ego4d-data.org/docs/start-here/#select-subset) has details on various subsets in the dataset.

The details of the PRE, PNR, and POST frames can be found in the annotations. Refer to this link (https://ego4d-data.org/docs/benchmarks/hands-and-objects/#annotations) for details on annotations for multiple hands-and-objects tasks.

Once you download the videos, you can use the annotations to get the PRE, PNR, and POST frames.

Let me know if there is any confusion!

Regards,
Siddhant Bansal

**Comment by idejie** [comment]
Thank you, @Sid2697 !

I found that the PRE, PNR, and POST frames have been provided in [annotations](https://ego4d-data.org/docs/benchmarks/hands-and-objects/#annotations) , with some keys: `clip_frame_number` or `video_frame_number`. 


BTW, is the index of the beginning frame of a video or clip `0` or `1`?

 I also found the adjacent frames of PNR are very similar to PNR. So how do you define or find the exact PNR frame during annotating? And for the PNR localization challenge, do we find the exact frame_number of PNR(same as the annotations)?

Best Wishes,
Dejie Yang

**Comment by Sid2697** [comment]
@idejie the index of the beginning frame is `0`.
Regarding annotations, we asked the annotators to label/select the PNR frames following the definitions from the paper.

Regards,
Siddhant Bansal

**Comment by idejie** [comment]
@Sid2697  Thanks a lot！

Best Wishes.
Dejie Yang

**Comment by Mr-Bigworth** [comment]
> 

Excuse me, if I want to get the training data about the Hands & Objects Benchmark. And which dataset should I choose? Annotations or clips? I'm not sure how to choose the Hands & Objects Only. 

ego4d --output_directory="~/ego4d_data" --datasets annotations --benchmarks HO
Is this correct?

**Comment by Mr-Bigworth** [comment]
> Hello @NNNNAI, thank you for reaching out!
> 
> Using the CLI, you'll be able to download the dataset and the corresponding annotations. This link (https://ego4d-data.org/docs/start-here/#select-subset) has details on various subsets in the dataset.
> 
> The details of the PRE, PNR, and POST frames can be found in the annotations. Refer to this link (https://ego4d-data.org/docs/benchmarks/hands-and-objects/#annotations) for details on annotations for multiple hands-and-objects tasks.
> 
> Once you download the videos, you can use the annotations to get the PRE, PNR, and POST frames.
> 
> Let me know if there is any confusion!
> 
> Regards, Siddhant Bansal

Excuse me, if I want to get the training data about the Hands & Objects Benchmark. And which dataset should I choose? Annotations or clips? I'm not sure how to choose the Hands & Objects Only.

ego4d --output_directory="~/ego4d_data" --datasets annotations --benchmarks HO
Is this correct?
